apps/emqx/include/logger.hrl

```diff
@@ -76,14 +76,17 @@
 -define(TRACE_FILTER, emqx_trace_filter).
 
 %% Only evaluate when necessary
+%% Always debug the trace events.
 -define(TRACE(Event, Msg, Meta),
     begin
     case persistent_term:get(?TRACE_FILTER, undefined) of
         undefined -> ok;
         [] -> ok;
         List ->
            emqx_trace:log(List, Event, Msg, Meta)
-    end
+    end,
+    ?SLOG(debug, (emqx_trace_formatter:format_meta(Meta))#{msg => Msg, trace_event => Event},
+        #{trace_filter => ignore})
```

Comment:
Suggestion: add a `trace_tag` field to `Meta` for both trace and log.
trace handler should only match logs with `trace_tag` field,
but do not inverse-match `trace_filter := ignore`.


```suggestion
    ?SLOG(debug, (emqx_trace_formatter:format_meta(Meta))#{msg => Msg, trace_tag => Event}, #{})
```

Reply:
I mean add it in this TRACE macro, but not where this macro is used.

Reply:
Sorry, trace_tag is in use and for both trace and log

maybe a is_trace=>true for traces?